disco_aws_automation/disco_metanetwork.py

```diff
@@ -236,3 +251,28 @@ def add_sg_rule(self, protocol, ports, sg_source=None, cidr_source=None):
         sg_args["to_port"] = ports[1]
         logging.debug("Adding sg_rule: %s", sg_args)
         self.vpc.vpc.connection.authorize_security_group(**sg_args)
+
+    def ip_by_offset(self, offset):
+        """
+        Pass in +10 and get 10th ip of subnet range
+        Pass in -2 and get 2nd to last ip of subnet
+
+        Returns IpAddress object, usually you'll want
+        to cast this to str.
+        """
+
+        try:
```

Comment:
Knowing that each subnet has 4096 IP addresses (/22), can we set the following restrictions:
```
offset > 2
offset % 4096 > 2 and offset % 4096 != 4095
offset < 12286
```
This is to avoid someone from using a network of broadcast IP address. I know I am going crazy, so feel free to discard this comment :)